packages/app/components/Settings/ChatbotQuestions.tsx

```diff
@@ -219,7 +294,7 @@ export default function ChatbotQuestions({
             View and manage the questions being asked of your chatbot
           </p>
         </div>
-        <Button onClick={() => setAddModelOpen(true)}>Add Question</Button>
+        {/* <Button onClick={() => setAddModelOpen(true)}>Add Question</Button> */}
```

Comment:
Remove it, if not used?

Reply:
temporarily disabled, but will be refactored to use later, so left it in for now for a reminder